generators/client/types.d.ts

```diff
@@ -1,23 +1,27 @@
 import type { addIconImport, addItemToMenu, addRoute } from '../angular/support/needles.js';
-import type { AngularApplication } from '../angular/types.js';
 import type { ExportApplicationPropertiesFromCommand } from '../../lib/command/index.js';
 import type { CypressApplication } from '../cypress/types.js';
 import type { JavaScriptApplication, JavaScriptSourceType } from '../javascript/types.js';
 import type { PostWritingEntitiesTaskParam } from '../../lib/types/application/tasks.js';
+import type { PartialAngularApplication } from '../angular/types-partial.js';
 import type Command from './command.ts';
 
 type ApplicationClientProperties = ExportApplicationPropertiesFromCommand<typeof Command>;
 
-export type ClientApplication = ApplicationClientProperties &
+export type FrontendApplication = ApplicationClientProperties &
   JavaScriptApplication &
-  AngularApplication &
   CypressApplication & {
     webappLoginRegExp: string;
     clientWebappDir?: string;
     webappEnumerationsDir?: string;
     clientFrameworkBuiltIn?: boolean;
   };
 
+/**
+ * Deprecated in favor of frontend application.
+ */
+export type ClientApplication = JavaScriptApplication & PartialAngularApplication & FrontendApplication;
```

Comment:
Types are experimental, there is no need to deprecated. Just drop it.

In any case a new issue should be created if we should keep Client rename everything to Frontend in v9. Same for server/backend.

Reply:
I do not really know if it should be deprecated (not enough refactoring  effect yet). Let's be a bit conservative in that regard and see later